<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>WithdrawMoney</class>
 <widget class="QDialog" name="WithdrawMoney">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>677</width>
    <height>317</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QDialog {
    background-color: #4a4a4a;
}

QStackedWidget {
    background-color: #4a4a4a;
    color: #ffffff;
}

QPushButton {
background-color: gray;
border-radius: 10px;
color:lightblue;
}

QLabel#label_main_menu,
QLabel#label_transfers,
QLabel#label_withdrawal_deposit {
    font: 20pt &quot;Arial&quot;;
    color: lightyellow;
}

QLabel {
color:lightblue;
font-family: Arial;
font-size: 12pt;
}

QLineEdit {
font-family: Arial;
font-size: 10pt;
background-color: lightgray;	
border: 1px solid white;
border-radius: 10px;
}

QTableView {
    background-color: #2e2e2e;
    color:lightblue;
border-radius: 10px;
}

QHeaderView::section {
    background-color: #1f1f1f;
	color:lightblue;
}

QTableView::item {
    border: 1px solid #3a3a3a;
}

QTableView::item:selected {
    background-color: #4a4a4a;
	color:lightblue;
 	gridline-color: gray
}
QHeaderView:vertical {
    background-color: #1f1f1f;
	color:lightblue;
    border: 1px solid #3a3a3a;
}</string>
  </property>
  <widget class="QStackedWidget" name="stackedWidget">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>681</width>
     <height>321</height>
    </rect>
   </property>
   <property name="currentIndex">
    <number>1</number>
   </property>
   <widget class="QWidget" name="page_choosing_amount_2">
    <widget class="QPushButton" name="btn_withdraw">
     <property name="geometry">
      <rect>
       <x>410</x>
       <y>160</y>
       <width>93</width>
       <height>29</height>
      </rect>
     </property>
     <property name="text">
      <string>Withdraw</string>
     </property>
    </widget>
    <widget class="QWidget" name="formLayoutWidget_2">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>120</y>
       <width>371</width>
       <height>31</height>
      </rect>
     </property>
     <layout class="QFormLayout" name="formLayout_2">
      <item row="0" column="0">
       <widget class="QLabel" name="label_4">
        <property name="text">
         <string>Amount</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLineEdit" name="txt_amount_2"/>
      </item>
     </layout>
    </widget>
    <widget class="QLabel" name="label_5">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>50</y>
       <width>471</width>
       <height>81</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Arial</family>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Withdraw an amount between 0 and 5000:</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_7">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>260</y>
       <width>101</width>
       <height>91</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="pixmap">
      <pixmap>../BankingAppImages/Logo_Def.png</pixmap>
     </property>
    </widget>
    <zorder>formLayoutWidget_2</zorder>
    <zorder>label_7</zorder>
    <zorder>label_5</zorder>
    <zorder>btn_withdraw</zorder>
   </widget>
   <widget class="QWidget" name="page_password_2">
    <widget class="QLabel" name="label_6">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>70</y>
       <width>349</width>
       <height>45</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Arial</family>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Enter password:</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="txt_password_2">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>120</y>
       <width>349</width>
       <height>26</height>
      </rect>
     </property>
     <property name="echoMode">
      <enum>QLineEdit::Password</enum>
     </property>
    </widget>
    <widget class="QPushButton" name="btn_deposit_money_2">
     <property name="geometry">
      <rect>
       <x>360</x>
       <y>150</y>
       <width>121</width>
       <height>29</height>
      </rect>
     </property>
     <property name="text">
      <string>Withdraw Money</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_8">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>260</y>
       <width>101</width>
       <height>91</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="pixmap">
      <pixmap>../BankingAppImages/Logo_Def.png</pixmap>
     </property>
    </widget>
    <zorder>label_8</zorder>
    <zorder>txt_password_2</zorder>
    <zorder>btn_deposit_money_2</zorder>
    <zorder>label_6</zorder>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
